Firmware/RTK_Surveyor/States.ino

```diff
@@ -232,6 +235,9 @@ void updateSystemState()
             ledcWrite(ledBTChannel, 0); //Turn off BT LED
           }
 
+          if (productVariant == REFERENCE_STATION)
```

Comment:
Help me understand what the statusLED on the ref station communicates. Is it similar to surveyor? Blinks while base is starting? PWM while in WiFi Config mode?

Reply:
A foot in the door... I was just replicating what the Surveyor does with its base LED - just as a place-holder. We can do something far more elegant with it for the Ref Stn...